src/Control/SAMLController.php

```diff
@@ -192,9 +191,9 @@ public function metadata()
                 header('Content-Type: text/xml');
                 echo $metadata;
             } else {
-                throw new \OneLogin_Saml2_Error(
+                throw new \OneLogin\Saml2\Error(
```

Comment:
Please import the class (`use OneLogin\Saml2\Error;`) and use the unqualified classname here.